cmd/zpool/zpool_main.c

```diff
@@ -1473,16 +1473,20 @@ static int
 max_width(zpool_handle_t *zhp, nvlist_t *nv, int depth, int max,
     int name_flags)
 {
-	char *name;
+	char *vdev_name;
 	nvlist_t **child;
 	uint_t c, children;
 	int ret;
 
-	name = zpool_vdev_name(g_zfs, zhp, nv, name_flags);
-	if (strlen(name) + depth > max)
-		max = strlen(name) + depth;
+	vdev_name = zpool_vdev_name(g_zfs, zhp, nv, name_flags);
+	max = MAX(max, strlen(vdev_name) + depth);
+	free(vdev_name);
 
-	free(name);
+	/*
+	 * vdev_name is "root-0" for the root vdev on my machine, but it's the
```

Comment:
What does this comment mean? `my machine`?

Reply:
On my machine all root vdev names are `root-0` (for all zpools).

I wasn't sure if it was possible for them to be something else (e.g. `root-1`, or `other-type-10`), but from a brief look at the codebase I now think root vdev names will *always* be `root-0`.

Code updated. If my understanding is incorrect, please let me know.